src/ui/common/src/components/pages/artifact/id/index.tsx

```diff
@@ -120,10 +120,12 @@ const ArtifactDetailsPage: React.FC<ArtifactDetailsPageProps> = ({
   }, []);
 
   useEffect(() => {
-    if (!!artifact && !sideSheetMode) {
-      document.title = `${
-        artifact ? artifact.name : 'Artifact Details'
-      } | Aqueduct`;
+    if (!!artifact) {
```

Comment:
how does any of this code impact the preview?

Reply:
if you look at few lines above, the actual fix is to remove the `!sideSheetMode` conditioning so we dispatch also in sidesheet

Reply:
but nnothing is dispatched in this block of code, it's just the setting of document.title. what am i missing here?

Reply:
nevermind, these github previews don't show the whole story.

okay i get it now :)

Reply:
yeah, that also make this kind of things harder to catch during code reviews